image/Dockerfile

```
@@ -8,10 +8,10 @@
 #    * [openapi2jsonschema](https://github.com/instrumenta/openapi2jsonschema)
 #
 # Golang is provided to allow running other Makefile targets
+FROM quay.io/kubevirtci/golang:v20250211-4e3c019
+ENV GIMME_GO_VERSION = 1.23.7
```

```suggestion
ENV GIMME_GO_VERSION=1.23.7
```

Subtle but I expect it to break with spaces.